src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,101 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline to be fulfilled. This deadline is a task that possesses a state/status that is by default
+ * not done.
+ */
+public class Deadline extends Task {
```

Comment:
`public class DeadlineTask extends Task {` is probably better, given the Naming standards for inherited classes. (You used it in `Command`, yourself.)